spikeinterface/sorters/runsorter.py

```diff
@@ -272,31 +281,34 @@ def run_sorter_container(sorter_name, recording, mode, container_image, output_f
     container_client.start()
 
     # check if container contains spikeinterface already
-    # this do not work with singularity:
-    # cmd = 'python -c "import spikeinterface; print(spikeinterface.__version__)"'
-    # this approach is better
-    cmd = ['python', '-c', 'import spikeinterface; print(spikeinterface.__version__)']
-    res_output = container_client.run_command(cmd)
+    cmd_1 = ['python', '-c', 'import spikeinterface; print(spikeinterface.__version__)']
+    cmd_2 = ['python', '-c', 'from spikeinterface.sorters import run_sorter_local']
+    res_output = ''
+    for cmd in [cmd_1, cmd_2]:
+        res_output += str(container_client.run_command(cmd))
     need_si_install = 'ModuleNotFoundError' in res_output
 
-
     if need_si_install:
+        # downgrade pip version to avoid dependency issues during installations
+        # see https://github.com/SpikeInterface/spikeinterface-dockerfiles/pull/14
+        cmd = 'pip install --no-input pip==21.2.4'
+        res_output = container_client.run_command(cmd)
```

Comment:
We should remove this. And put it in every docker that need it.

Reply:
Ok, tell me when I can check the updated docker container.